src/utils/helpers/validatePage.tsx

```diff
@@ -0,0 +1,113 @@
+import {
+  ApplicationElementStates,
+  ElementState,
+  ProgressStatus,
+  ResponsesFullByCode,
+  ResponseFull,
+} from '../../utils/types'
+
+import { TemplateElementCategory } from '../../utils/generated/graphql'
+
+export const PROGRESS_STATUS: { [key: string]: ProgressStatus } = {
+  NOT_VALID: 'NOT_VALID',
+  VALID: 'VALID',
+  INCOMPLETE: 'INCOMPLETE',
+}
+
+export const getCombinedStatus = (
+  array: { status: ProgressStatus }[] | undefined,
+  strictly: boolean = false
```

Comment:
what is strictly for this method ?

Reply:
I was reconsidering this one. I think this is not needed anymore. I was thinking we would show the section state (combined of pages statuses) different when we ran the validation. After our meetings and the document that @CarlosNZ created that the combined status would always return:
> Any INVALID page in Section → Section state = ❌
> Any NEUTRAL page in Section → Section state = Neutral
> ELSE:  Section state = ✅